### ISSUE TITLE
[Example] Nuxt Product Gallery

### ISSUE THREAD

**Issue by colbyfayock** [cloudinary-community/cloudinary-examples#48]
Create a new example showing how to set up the Cloudinary Product Gallery in an Nuxt project.

See other frameworks for examples:
* https://github.com/cloudinary-community/cloudinary-examples/tree/main/examples/react-product-gallery

**Comment by colbyfayock** [comment]
yeah no worries - did you just want to abandon the other PR though? seemed like we were close! i'll assign this for now to you though

**Comment by colbyfayock** [comment]
@sccalabr you would need to implement this manually using the script tag and Javascript

here's an example in Vue that I assume would be similar but different enough given extra features Nuxt has to improve the developer experience:

https://github.com/cloudinary-community/cloudinary-examples/blob/main/examples/vue-product-gallery/src/components/CldProductGallery.vue

**Comment by colbyfayock** [comment]
all good, thanks for the heads up!

**Comment by error9098x** [comment]
@colbyfayock Hi, how are you doing, I wanted to know if I can continue to add some examples here under hacktoberfest. Is this repo taking part in hacktoberfest 2024?

**Comment by colbyfayock** [comment]
hi @error9098x, as of now, PRs in this repository wouldn't quality for Cloudinary Hacktoberfest swag, as we're hoping to get more contributions to our SDKs for the limited amount of swag we're giving out. perhaps there are some interesting Issues open between all of those? In particular with nuxt: https://github.com/nuxt-modules/cloudinary/issues

this might change later in the month depending on the amount of contributions we get
